virt/install/installing-ove.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="installing-ove"]
+= Installing an {ove} cluster
+:context: installing-ove
+
+toc::[]
+
+The following procedures describe a streamlined method to install a {ove-first} cluster that requires less planning and configuration compared to other installation methods.
+
+:FeatureName: Installing an {ove} cluster using this method
+include::snippets/technology-preview.adoc[]
+
+// Is it bad to mix mentions of OVE above and plain Virt below?
+
+This method has the following advantages for users who want to install a cluster primarily to run virtualized workloads using {VirtProductName}:
+
+* The cluster installation is performed through a graphical user interface.
+
+* You can install the Virtualization bundle, which includes all of the additional Operators needed to run virtual machines on your cluster, at the same time as the cluster installation.
+
+* If you are installing your cluster in a disconnected environment, you do not need to configure an image registry in your environment when using this method.
+
+// Probably need some section for limitations such as bare metal machines, unless that's a commonly understood requirement for virt.
+
+// Downloading the installation ISO
+include::modules/installing-ove-iso.adoc[leveloffset=+1]
+
+// Mounting the ISO and booting the machines
+include::modules/installing-ove-booting.adoc[leveloffset=+1]
+
+// Configuring the cluster installation from the web console
+include::modules/installing-ove-console.adoc[leveloffset=+1]
+
+// Should I include the Agent procedure here about tracking and verifying installation progress? Or does the UI do the same thing as the `wait-for` commands?
+
+After the cluster has been installed, the installation console displays the URL of the web console, which you can visit and sign in using your downloaded credentials.
+
+[id="next-steps_{context}"]
+== Next steps
+
+After a successful cluster installation, you can take the following next steps to prepare your cluster to run virtualized workloads:
```

Comment:
I'm not an expert here, so leaving to @tiraboschi and @dominikholler for more useful comments :) .
Btw, not sure if you could directly point to some already existing docs for OVE, though

Reply:
Happy to add or remove any links in this section. That said, I think the [OVE docs](https://docs.redhat.com/en/documentation/red_hat_openshift_virtualization_engine/4) are a little bare at the moment.